completion/git-machete.fish

```diff
@@ -136,6 +143,7 @@ complete -c git -n "__fish_git_using_command machete; and __fish_seen_subcommand
 # git machete slide-out
 complete -c git -n "__fish_git_using_command machete; and not __fish_seen_subcommand_from $__mcht_commands" -f -a slide-out -d 'Slide out the current branch and sync its downstream (child) branches with its upstream (parent) branch via rebase or merge'
 complete -c git -n "__fish_git_using_command machete; and __fish_seen_subcommand_from slide-out; and not __fish_seen_subcommand_from --merge -M --down-fork-point -d"                                                        -x -l down-fork-point -s d  -d 'If updating by rebase, specifies the alternative fork point for downstream branches for the operation. git machete fork-point overrides for downstream branches are recommended over use of this option. See also doc for --fork-point option in git machete help reapply and git machete help update. Not allowed if updating by merge'
+complete -c git -n "__fish_git_using_command machete; and __fish_seen_subcommand_from slide-out; and not __fish_seen_subcommand_from --delete"                                                                               -f -l delete                -d 'Delete branches from git'
 complete -c git -n "__fish_git_using_command machete; and __fish_seen_subcommand_from slide-out; and not __fish_seen_subcommand_from --merge -M"                                                                             -f -l merge           -s M  -d 'Update the downstream branch by merge rather than by rebase'
```

Comment:
Nit: IIUC how it works, `--down-fork-point -d` should be added to the final `__fish_seen_subcommand_from --merge -M`

Reply:
Coz `-d` isn't allowed together with `-M` (and line 145 correctly reflects that)